### ISSUE TITLE
Invisibile ELO columns on hero-page

### ISSUE THREAD

**Issue by ThePianoDentist** [datdota/datdota#15]
![image](https://user-images.githubusercontent.com/17036692/33530596-13ac7cac-d882-11e7-93f6-e46ac4ea1f51.png)

http://datdota.com/heroes/elo?default=true

Mozilla/5.0 (X11; Linux x86_64; rv:58.0) Gecko/20100101 Firefox/58.0

at 1280 X 800

**Comment by Noxville** [comment]
Changed how this reacts responsively to small screens - should be good now.

**Comment by davidfischer** [comment]
I realized this just got closed but I had taken a look at this so I'll just drop my notes in in case they prove useful.

I created a gist illustrating the changes:
https://gist.github.com/davidfischer/737fba561c607c1287c92b846290f362

Basically, I'm proposing that any viewport less than 1200px doesn't see 3 columns side by side but rather they stack. If the screen is wider than 1200px I left it 3 columns. The second change is to remove [`responsive: true`](https://datatables.net/reference/option/responsive) from the data tables initialization and switch to `false`. Setting responsive causes columns to get hidden when space is tight. Setting to false introduces horizontal scrolling when necessary which isn't ideal but I think it's the lesser of two evils.
